### PR TITLE
Select time screen

### PR DESCRIPTION
**Pull request by iameru**
This is the PR for issue #41 

this is a branch off from select-time-loader #46 , I will take care of merging changes until this PR is merged.

### CODE REVIEW

**Comment by iameru** [comment]
ready for review. 
Some comments: 
1. Im not sure about the usage of the `form` element here but thought its semantically better than any other option
2. I didnt write the final navigate to the selecting seats as this is not implemented yet
3. the button component I should have used was also not ready yet

So shall I wait for both issues to be resolved or rather remember to reopen later on again?

**Comment by iameru** [comment]
draft again as I misunderstood the task and created the time and date selection elements as radio-inputs. will talk with @acz1992 to resolve this and implement the stuff they did

File: src/components/SeatSelectButton.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  selected,
+  occupied,
+  onClick,
+}: SeatSelectButtonProps) {
+  return (
+    <button
+      onClick={onClick}
+      disabled={occupied}
+      className={clsx(
+        "rounded h-[28px] aspect-square disabled:bg-dark-light",
```
Comment (dnmct) [inline src/components/SeatSelectButton.tsx:21]: there is a tailwind size for that height dont use Arbitrary values when you dont have to

File: tailwind.config.cjs [hunk 2]
```diff
@@ -21,6 +21,7 @@ module.exports = {
       },
       red: "#EF4444",
       green: "#22C55E",
+      none: "00FFFFFF",
```
Comment (dnmct) [inline tailwind.config.cjs:24]: css value for that is transparent if that what you want to achieve

File: src/components/SeatSelectButton.tsx (hunk 1 above)
Comment (iameru) [reply src/components/SeatSelectButton.tsx:21]: This is related to  #62 which this is a branch off, not changing but will merge changes made there in my PR here.

File: tailwind.config.cjs (hunk 2 above)
Comment (iameru) [reply tailwind.config.cjs:24]: This is related to  #62 which this is a branch off, not changing but will merge changes made there in my PR here.

**Comment by iameru** [comment]
to clarify: I merged #62 into here to use the component. Will merge changes of it into here in case its needed - however the code looks as if this PR could get merged to main and the button could follow

File: src/routes/SelectTime.tsx [hunk 3]
```diff
@@ -44,13 +47,88 @@ export async function selectTimeLoader(): Promise<DatesType> {
...
+      className="flex flex-col justify-between h-full"
+      onSubmit={(event) => {
+        event.preventDefault();
+        // Navigate to seats selection when it is done
+      }}
+    >
+      <div className="flex flex-col gap-6">
+        <div className="space-y-5">
+          <h3 className="font-bold text-sm text-white-dimmed">DATE</h3>
+          <div className="grid grid-cols-4 grid-rows-[2rem_2rem_2rem_2rem] gap-y-2 gap-x-1">
```
[the remark this answers is not in the document]
Comment (iameru) [reply src/routes/SelectTime.tsx:67]: As this is the only place I see a grid with these 4 row heights I figured its overthinking to abstract this into the config.
If you dont follow this reasoning I will do it if you want me to, tell me.

File: src/routes/SelectTime.tsx [hunk 4]
```diff
@@ -44,13 +47,88 @@ export async function selectTimeLoader(): Promise<DatesType> {
...
   };
   return (
-    <div>
-      <DetailHeader {...detailHeaderProps} />
-    </div>
+    <form
+      className="flex flex-col justify-between h-full"
+      onSubmit={(event) => {
+        event.preventDefault();
+        // Navigate to seats selection when it is done
```
[the remark this answers is not in the document]
Comment (iameru) [reply src/routes/SelectTime.tsx]: fixed in 
21a61e8

File: src/routes/SelectTime.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (franzwollang) [reply src/routes/SelectTime.tsx:69]: Tailwind already has preset values for grid rows that should be more than enough for this use case. You shouldn't need to change the config.

[the remark this answers is not in the document]
Comment (iameru) [reply src/routes/SelectTime.tsx]: fixed height was indeed not needed anymore 
a16235e

**Pull request merged**